### PR TITLE
Heading elements

### PR DESCRIPTION
**Pull request by jrgarou**
This PR adds:
- `h1`, `h2`... to all pages, creating a heading hierarchy structure.
- `<html lang="pt-br">`.
-  `components/listings/shared/Benefits` to be shared with home and seller pages. With [its amplitude logs](https://github.com/emcasa/frontend/blob/167d44b0b79821819f083d58daef0abe41bc350a/lib/amplitude.js).
- Add Custom `/imoveis/...` titles

https://www.notion.so/emcasa/Heading-elements-2568eb1736064d489cb77e50907a0517

### CODE REVIEW

File: components/listings/shared/ListingList/title.js [hunk 1]
```diff
@@ -1,17 +1,245 @@
...
+    city: 'rio-de-janeiro',
+    value: 'na Zona Sul do Rio de Janeiro'
+  },
+  {
+    state: 'sp',
+    value: 'em São Paulo'
+  }
+]
+
+function getTitleTextByFilters(neighborhoodsSlugs) {
```
Comment (lucas-tulio) [inline components/listings/shared/ListingList/title.js]: Acho que vai precisar atualizar os testes dessa função.

File: components/listings/sell/HowItWorks/styles.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+`
+
+export const Content = styled(View)`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  max-width: 1100px;
+  @media (max-width: ${theme.breakpoints[0]}) {
+    padding-left: 20px;
+    padding-right: 20px;
```
Comment (lucas-tulio) [inline components/listings/sell/HowItWorks/styles.js]: Aqui a gente pode usar as medidas do tema (`...space[4]`, etc).

File: components/listings/shared/ListingList/title.js (hunk 1 above)
Comment (jrgarou) [reply components/listings/shared/ListingList/title.js]: Atualizei os testes.

File: components/listings/sell/HowItWorks/styles.js (hunk 2 above)
Comment (jrgarou) [reply components/listings/sell/HowItWorks/styles.js]: Updated! 👍

**Pull request merged**